Metadata/MetadataConverter.php

```diff
@@ -63,7 +63,22 @@ public function convert(ClassHierarchyMetadata $metadata)
             if (null !== $classMetadata->arguments) {
                 $definition->setArguments($classMetadata->arguments);
             }
+            if (null !== $classMetadata->autowire) {
+                if (!method_exists($definition, 'setAutowire')) {
+                    throw new InvalidAnnotationException(
+                        sprintf(
+                            "autowiring require symfony >=2.8 on class %s"
```

Comment:
Sounds good!
I based it on https://github.com/schmittjoh/JMSDiExtraBundle/pull/254/files#diff-795eb25e349a10927c913d862c5cf87aL73 I will fix this other one too.